### DISCUSSION TITLE
PIMD 3.0 release date

### DISCUSSION THREAD

**Discussion by SaratVajrapu** [troglobit/pimd#194] [General]
Hi,
I am exploring the pimd module to use in one of my projects and noted that the last release is pimd-2.3.2 which does not include pimctl and lot of bug fixes went after that. Just curious to know when is the next release planned so that I can use the official release version instead of beta.

**Comment by troglobit** [comment]
This is a volunteer based open source project, so we cannot commit to any timelines.  It is however more likely the sister project pimd-dense will see a release before this one.

**Reply by SaratVajrapu** [reply]
Thanks for the update..
